tests/Behat/Tests/Config/SuiteTest.php

```diff
@@ -61,4 +65,18 @@ public function testAddingPaths(): void
             ]
         ], $config->toArray());
     }
+
+    public function testAddingFilters(): void
```

Comment:
You are missing the test to confirm that you cannot repeat a filter

Reply:
done